actionview/lib/action_view/template.rb

```diff
@@ -256,8 +256,14 @@ def encode!
     end
 
     # This method is responsible for marking a template as having strict locals
-    # and extracting any arguments declared in the format
-    # locals: (message:, label: "My Message")
+    # which means the template can only accept the locals defined in a magic
+    # comment. For example, if your template acceps that locals +title+ and
```

Comment:
I think this is supposed to be:

```suggestion
    # comment. For example, if your template accepts the locals +title+ and
```

Reply:
Thanks, fixed in bd7b6decb98827934ccbd53b27dc1ec997479332

Reply:
Looks like that fix still has `acceps`.